src/Fabulous/Builders.fs

```diff
@@ -7,7 +7,7 @@ open Fabulous.StackAllocatedCollections
 open Fabulous.StackAllocatedCollections.StackList
 open Microsoft.FSharp.Core
 
-type AttributesBundle = (struct (StackList<ScalarAttribute> * WidgetAttribute[] voption * WidgetCollectionAttribute[] voption * EnvironmentAttribute[] voption))
```

Comment:
I like that we can remove `ValueSome and ValueNone` when using it. Only downside is that we will need to update the `Avalonia and MauiControls` in many places :(